Updated tests

lee-delarm6 opened the pull request:
Updated tests

## Summary

- Improved test targetting and updated text check for new section
- This only affects tests

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/64962

## Testing done

- Updated test data
- Ran and verified new tests run properly

## Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).

lee-delarm6 committed:
Updated tests

Updated tests

lee-delarm6 committed:
Merge branch 'main' into 64962_improve_cypress_tests

wullaski commented on src/applications/check-in/tests/e2e/pages/NextOfKin.js:
```diff
@@ -49,16 +49,22 @@ class NextOfKin {
-      );
+      this.openAdditionalInfo();
+      cy.get('div[data-testid="additional-info"]')
+        .should('be.visible')
```
For the `be.visible` asserts we should check the visibility on a div or something within it that is visible after you open the drawer.

lee-delarm6 replied:
True, gotcha

wullaski reviewed (changes requested):
There's an opportunity here to add another file that tests the next of kin display on day of and uses the dayOf method to test for the correct text. We could also test the other things that the `src/applications/check-in/pre-check-in/tests/e2e/next-of-kin-display/next.of.kin.display.cypress.spec.js` file tests on the day-of app.

lee-delarm6 committed:
Updated check inside additional info

Updated check inside additional info

lee-delarm6 committed:
Merge branch '64962_improve_cypress_tests' of https://github.com/department-of-veterans-affairs/vets-website into 64962_improve_cypress_tests

lee-delarm6 committed:
Added visibility check closed

Added visibility check closed, eslint disable on import that doesn't exist elsewhere

lee-delarm6 committed:
Merge branch 'main' into 64962_improve_cypress_tests

lee-delarm6 commented:
All items addressed now 👍

wullaski reviewed (approved):
LGTM!
